packaging/linux/systemd/kbfs.service

```diff
@@ -11,8 +11,9 @@ Type=notify
 # Without this line, `systemctl --user restart kbfs.service` will hit mount
 # failures if there are any running shells cd'd into a Keybase folder.
 ExecStartPre=-/usr/bin/env fusermount -uz /keybase
-ExecStart=/usr/bin/kbfsfuse -debug -log-to-file /keybase
+ExecStart=/usr/bin/kbfsfuse -debug -log-to-file $KEYBASE_MOUNTDIR
```

Comment:
A couple ideas I'd like to consider here, to make it possible for `kbfs.service` to boot on its own without `run_keybase`:

1. We could move the `${XDG_DATA_HOME:-$HOME/.local/share}` logic from `run_keybase` down to here. Systemd doesn't support fancy Bash variable expansion directly, but I think it's somewhat common to get at it by invoking bash, e.g.

    ```
    ExecStart=/usr/bin/bash -c '/usr/bin/kbfsfuse -debug -log-to-file "${XDG_DATA_HOME:-$HOME/.local/share}/keybase/fs"'
    ```

    ...Or even give up on `XDG_DATA_HOME` entirely, since it's so rarely set in practice, if that's the only thing that's forcing us to capture the calling environment?

2. We could move configuration for this onto the filesystem more directly. For example, maybe we could say that `~/.config/keybase/kbfs_mount` contains a path, and both `keybase` and `kbfsfuse` (when not given a positional mountpoint, or maybe when given a new `-read-mount-path` flag) read that file and fall back to a default if it's missing. I'd almost suggest that `mount_point` could be a symlink, except that I bet that gets weird on Windows.

Anyway, I'm willing to accept that my worries here are pretty niche, and that no one else will notice or care, and that making either of these changes is more trouble than it's worth. So I'm gonna go ahead and approve the PR as is, and let you make the judgment call :)

Reply:
My call is to go with door number 2, except that we can use the existing `~/.config/keybase/config.json` file, which already has an option for "mountdir" that is read by the usually libkb env logic.  It will still require you to run `run_keybase` to set up that config option at first, and it won't support multiple instances of kbfs running under systemd, but I think that's ok.